omod/src/main/java/org/openmrs/module/referenceapplication/page/controller/LoginPageController.java

```diff
@@ -110,11 +114,24 @@ public String get(PageModel model,
 			Context.removeProxyPrivilege(GET_LOCATIONS);
 		}
 
+        model.addAttribute("showSessionLocations", !isLocationUserPropertyAvailable(administrationService));
```

Comment:
Did you intentionally duplicate the model.addAttribute call?

Reply:
Nope @dkayiwa, It used to set that value always, and if wanted it will update this value by next line